test/tools/integration/Makefile

```diff
@@ -51,7 +51,6 @@ plan apply destroy: .terraform
 		-var hcloud_test_server_name="machine-controller-test-$(BUILD_ID)"
 
 provision: apply
-	cd ../verify/ && make verify
-	cd ../../../ && make all
+	make -C ../../../ clean
+	make -C ../../../ all GOOS=linux
 	./provision_master.sh
-	./create-and-destroy-machine.sh
```

Comment:
I guess that this script doesn't exist anymore and that's why you dropped it here. Is that correct or there's some other reason?

Reply:
Yep. This Makefile is not used in the e2e and it was broken. I fixed it to run the tests locally.